.gitignore

```diff
@@ -10,3 +10,12 @@
 /build/linux/arch/src
 /build/linux/arch/*.tar.gz
 /build/linux/arch/*.tar.xz
+.idea/workspace.xml
+.idea/vcs.xml
+.idea/modules.xml
+.idea/misc.xml
+.idea/ledger-live-desktop.iml
+.idea/inspectionProfiles/Project_Default.xml
+.DS_Store
+.gitignore
+yarn.lock
```

Comment:
Remove all those lines :)
- consider using global gitignore for editor/environment/os specific files
- .gitignore and yarn.lock shouldn't be ignored